api/include/opentelemetry/trace/default_tracer.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+#include "opentelemetry/nostd/unique_ptr.h"
+#include "opentelemetry/trace/default_span.h"
+#include "opentelemetry/trace/span.h"
+#include "opentelemetry/trace/tracer.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace trace
+{
+class DefaultTracer : public Tracer
+{
+public:
+  ~DefaultTracer() = default;
+
+  /**
+   * Starts a span.
+   *
+   * Optionally sets attributes at Span creation from the given key/value pairs.
+   *
+   * Attributes will be processed in order, previous attributes with the same
+   * key will be overwritten.
+   */
+  nostd::unique_ptr<Span> StartSpan(nostd::string_view name,
+                                    const KeyValueIterable &attributes,
+                                    const StartSpanOptions &options = {}) noexcept
```

Comment:
nitpik: override this, and below two functions.

Reply:
Same as above